### ISSUE TITLE
Describe API and limitations

### ISSUE THREAD

**Issue by wmertens** [reactabular/easy#3]
It is nice that `easy` combines everything needed, but you actually need to go in and do a real deep dive to know what the API is and what everything does. It would be great if the exposed API was documented, and also a caveat section providing some examples of when it would be better to use the reactabular components separately.

**Comment by bebraw** [comment]
Yeah, that's a good idea. I actually pushed a big chunk of that problem to [column extensions](https://github.com/reactabular/column-extensions).
